Arista/ConfigFiles/telegraf-default.conf

```diff
@@ -186,4 +186,15 @@
   # ssl_cert = "/etc/telegraf/cert.pem"
   # ssl_key = "/etc/telegraf/key.pem"
   ## Use SSL but skip chain & host verification
-  # insecure_skip_verify = false
\ No newline at end of file
+  # insecure_skip_verify = false
+
+[[inputs.bond]]
+  ## Sets 'proc' directory path
+  ## If not specified, then default is /proc
+  # host_proc = "/proc"
+
+  ## By default, telegraf gather stats for all bond interfaces
+  ## Setting interfaces will restrict the stats to the specified
+  ## bond interfaces.
+  # bond_interfaces = ["bond0"]
```

Comment:
I think you meant this to be uncommented?

Reply:
on most of the user servers the bond interface is named as macvlan-bond0

Reply:
no, default config is good enough for our case. It will collect from all bond interfaces. 

what is macvlan-bond0?
I only see bond0  on us180:
cat /proc/net/bonding/bond0

Reply:
ok.